### PR TITLE
Updating Documentation

### PR DESCRIPTION
**Pull request by deadex-ng**
I've made the PR to preview doc builds.

### CODE REVIEW

File: docs/img/.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (theblockstalk) [inline docs/img/.yml:1]: remove this file

File: docs/architecture.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Blockchain Client
+-----------------------
+The blockchain client connects the SDK to our blockchain service that is running two smart contracts, eosio and id.tonomy. eosio is a system level governance 
+contract that sets who is allowed to run the blockchain and create accounts. 
+
+Additionally, id.tonomy is a smart contract that manages permissions for users to create and login into accounts. It also provides additional cryptographic information for the SDK logic to work. 
+
+We call the getApi() function in this `file <https://github.com/Tonomy-Foundation/Tonomy-ID-SDK/blob/development/src/services/eosio/eosio.ts>`_ to access the Blockchain client.
+
+crypto
```
Comment (theblockstalk) [inline docs/architecture.rst:35]: good descriptions generally!

**Review by theblockstalk — COMMENTED** [review]
looks good. please export the Tonomy ID SDK image directly from
https://drive.google.com/drive/folders/1VWZqUdcaBvMhWd-yXmUyfUOJyLYdvIvu

File: docs/img/.yml (hunk 1 above)
Comment (deadex-ng) [reply docs/img/.yml]: I've removed it

File: docs/architecture.rst (hunk 2 above)
Comment (deadex-ng) [reply docs/architecture.rst:35]: Thanks @theblockstalk

**Pull request merged**